client/my-sites/upgrades/domain-management/edit/mapped-domain.jsx

```diff
@@ -110,10 +111,11 @@ const MappedDomain = React.createClass( {
 
 		return (
 			<VerticalNavItem path={ path }>
-				{ this.translate( 'DNS Records' ) }
+				{ this.props.translate( 'DNS Records' ) }
 			</VerticalNavItem>
 		);
 	}
 } );
 
-module.exports = MappedDomain;
+export { MappedDomain };
```

Comment:
We test the pure components, because it makes things much easier to test, and we're not interested in testing the whole system together (we have full e2e tests for that) just the behavior of that UI component. I think http://ericnish.io/blog/how-to-unit-test-react-redux-components/ is a decent write up on the reasoning behind this. 

If folks think this test is not useful, consider removing it and maybe replacing it with a cleaner unit test, or a new e2e test in https://github.com/Automattic/wp-e2e-tests